### PR TITLE
Implement the rest of the models and update Db.cs

### PR DESCRIPTION
**Pull request by toplenboren**
Add CORS headers, Implement the rest of the models and update Db.cs

@usernamedt I need help with FluentAPI and model relation implementation

I suppose, to define OTM and MTM relationships on models like Moderators or smth like this we should do something like that for all models 😮 
```
 modelBuilder.Entity<TeamUser>()
                .HasKey(k => new { k.TeamId, k.UserId });

            modelBuilder.Entity<TeamUser>()
                .HasOne(u => u.Team)
                .WithMany(m => m.TeamUsers)
                .HasForeignKey(u => u.TeamId)
                .IsRequired();

            modelBuilder.Entity<TeamUser>()
                .HasOne(u => u.User)
                .WithMany(m => m.TeamUsers)
                .HasForeignKey(u => u.UserId)
                .IsRequired();
```

I don't want to write a lot of code in `Db.cs` file, especially I don't want to write copy paste code. Is there a way to make my life easier?

### CODE REVIEW

**Comment by toplenboren** [comment]
Also I need to add migrations

File: Quest.Domain/Enums/TaskAttemptStatus.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿namespace Quest.Domain.Enums
+{
+    public enum TaskAttemptStatus
+    {
+        Error = 0,
+        Accept = 1,
```
Comment (usernamedt) [inline Quest.Domain/Enums/TaskAttemptStatus.cs]: Accepted

**Comment by toplenboren** [comment]
Help gotten, need review @usernamedt ^^

**Review by usernamedt — COMMENTED** [review]
refactor needed

File: Quest.Domain/Enums/TaskAttemptStatus.cs (hunk 1 above)
Comment (toplenboren) [reply Quest.Domain/Enums/TaskAttemptStatus.cs]: Have changed

**Comment by toplenboren** [comment]
@usernamedt was it the one and only refactor needed? Requesting re-review

**Pull request merged**